python/tskit/drawing.py

```diff
@@ -117,6 +122,20 @@ def check_x_scale(x_scale):
     return x_scale
 
 
+def check_ticks(ticks, default_iterable):
+    """
+    If y_ticks is iterable, it is a list of tick positions. Otherwise return a default
+    or None if falsey
+    """
+    if ticks is None:
+        return default_iterable
+    try:
+        iter(ticks)
+        return ticks
+    except TypeError:
+        raise NotImplementedError("Autocalculated tick mark locations not implemented.")
```

Comment:
This is still the wrong error class and message. Just return ``ticks`` and let the standard errors do the work.

Reply:
Weird. I did change that, and it's not what's now on my branch: https://github.com/hyanwong/tskit/blob/2382583b1f72de16646637aec9255c5bb0c01744/python/tskit/drawing.py#L125. Not sure what's up here @jeromekelleher